app/models/manageiq/providers/vmware/manager_auth_mixin.rb

```diff
@@ -28,21 +28,23 @@ def verify_credentials(auth_type = nil, options = {})
   def connect(options = {})
     raise "no credentials defined" if missing_credentials?(options[:auth_type])
 
-    server   = options[:ip] || address
-    port     = options[:port] || self.port
-    username = options[:user] || authentication_userid(options[:auth_type])
-    password = options[:pass] || authentication_password(options[:auth_type])
+    server      = options[:ip] || address
+    port        = options[:port] || self.port
+    api_version = options[:api_version] || self.api_version
+    username    = options[:user] || authentication_userid(options[:auth_type])
+    password    = options[:pass] || authentication_password(options[:auth_type])
 
-    self.class.raw_connect(server, port, username, password)
+    self.class.raw_connect(server, port, username, password, api_version)
   end
 
   module ClassMethods
-    def raw_connect(server, port, username, password, validate = false)
+    def raw_connect(server, port, username, password, api_version = '5.5', validate = false)
```

Comment:
If you change the args to `raw_connect` you'll need to update the UI task args here https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/mixins/ems_common_angular.rb#L124

Reply:
😱 Glad to have such a great reviewer, thank you so much, everything would break. Here's the PR https://github.com/ManageIQ/manageiq-ui-classic/pull/3701, best to merge both at same time.

Reply:
It was top-of-mind because @slemrmartin is looking at improving the way we validate credentials for exactly this reason :smile: